Misc/NEWS.d/next/Core and Builtins/2019-09-15-01-30-06.bpo-35696.1iK_1H.rst

```diff
@@ -0,0 +1 @@
+Slightly improve perfomance of ``long_compare()``. Patch by hongweipeng and Sergey Fedoseev.
```

Comment:
Honestly, I don't think that the optimization is significant enough to be mentioned. It's just 1 nanosecond on a microbenchmark. Nobody will notice it on a macrobenchmark. Usually, we don't document such micro-optimization.

Reply:
I've remove news and changed the title to replace "improving performance" with "removing unnecessary...". Thank you for your review.